accelerators/federated-learning/automated_provisioning/ReadMe.md

```diff
@@ -0,0 +1,107 @@
+- `SubscriptionId`: the Id of the subscription where the silo will be created (default: "48bbc269-ce89-4f6f-9a12-c6f91fcb772d", _a.k.a._ the AIMS subscription). 
+- `K8sClusterName`: the name of the K8s cluster to be created (default: "cont-k8s-01"). It will live in a resource group named like the cluster, with "-rg" appended.
+- `RGLocation`: the location of the K8s cluster and its corresponding resource group (default: "westus2").
+- `AgentCount`: the number of agents in the K8s cluster (default: 1 - beware, this should be an _int_, not a _string_).
+- `AgentVMSize`: The agent VM SKU (default: "Standard_B4ms").
+
+The command should look something like the below (with the parameters replaced by your own values of course).
+
+```ps1
+./ps/CreateK8sCluster.ps1 "Your-Silo-SubscriptionId" "Name-of-Cluster-to-Create" "Location-of-Cluster-to-Create" Number-of-Agents "VM-SKU"
+```
+
+### Connect it to the orchestrator workspace
+
+To connect a silo to an orchestrator Azure ML workspace, the following needs to happen:
+- create the Azure ML workspace if it doesn't already exist;
+- connect the K8s to Azure Arc; 
+- deploy the Azure ML extension on the Arc cluster;
+- attach the Arc cluster to the Azure ML workspace.
+
+Here again, there is a script that does all of that for you: `ConnectSiloToOrchestrator.ps1`. It takes the following input arguments.
+- `SubscriptionId_Orchestrator`: the Id of the subscription to which the orchestrator will belong (default: "48bbc269-ce89-4f6f-9a12-c6f91fcb772d", _a.k.a._ the AIMS subscription).
+- `AMLWorkspaceName`: the name of the orchestrator Azure ML workspace to create, if it doesn't exist already (default: "aml1p-ml-wus2").
+- `AMLWorkspaceRGName`: the name of the orchestrator resource group to create, if it doesn't exist already (default: "aml1p-rg").
+- `AMLWorkspaceLocation`: the location of the orchestrator Azure ML workspace (default: "westus2").
+- `K8sClusterName`: the name of the K8s cluster to connect to the orchestrator Azure ML workspace (default: "cont-k8s-01"). **Note that this is just used to create the name of the Arc cluster and its resource group. The K8s cluster is referenced implicitly by the kube config file that was created during the previous step.**
+- `AMlComputeName`: the name of the Azure ML compute to be created (default: "cont-01-compute" - must be between 2-16 characters and only contain alphanumeric characters or dashes). **This is the compute name you will be using when submitting jobs.**
+
+The command should look something like the below (with the parameters replaced by your own values of course).
+
+```ps1
```

Comment:
Nit: `ps1` &mapsto; `ps` or `powershell` here and in the other places you type sample PowerShell commands, i.e.

```markdown
    ```powershell
    ./ps/ConnectSiloToOrchestrator.ps1  "..."
    ```
```